src/store/customer/actions.js

```diff
@@ -76,8 +90,10 @@ export async function loginFb({commit}) {
       _procError(err)
     })
 }
+
 export async function registerFb({commit}) {
-  var user = await getUserFbInfo()
+  let token = getFbToken(),
```

Comment:
2 functions:  getUserFbInfoByToken , getUserFbInfo
could be merge into 1 function: getUserFbInfo  (with getFbToken() checking inside)
--> coz the final point is to get User's FbInfo